Source/Menu/DownloadContentForm.cs

```diff
@@ -0,0 +1,1270 @@
+
+                    if (MessageBox.Show(message, Catalog.GetString("Attention"), MessageBoxButtons.YesNo, MessageBoxIcon.Warning) != DialogResult.Yes)
+                    {
+                        // cancelled
+                        EnableButtons();
+                        return;
+                    }
+                }
+            }
+            catch (System.IO.DriveNotFoundException)
+            {
+                message = Catalog.GetStringFmt("Drive not available");
+                MessageBox.Show(message, Catalog.GetString("Attention"), MessageBoxButtons.OK, MessageBoxIcon.Error);
+                // cancelled
+                EnableButtons();
+                return;
+            }
+
+            if (!Directory.Exists(installPathRoute))
+            {
+                try
+                {
+                    Directory.CreateDirectory(installPathRoute);
+                }
+                catch (Exception createDirectoryException)
+                {
+                    message = Catalog.GetStringFmt("Directory \"{0}\" cannot be created: {1}",
+                        installPathRoute, createDirectoryException.Message);
+                    MessageBox.Show(message, Catalog.GetString("Attention"), MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    EnableButtons();
+                    return;
+                }
+            }
+            else
+            {
+                if ((Directory.GetFiles(installPathRoute).Length != 0) ||
+                    (Directory.GetDirectories(installPathRoute).Length != 0))
+                {
+                    message = Catalog.GetStringFmt("Directory \"{0}\" exists and is not empty", installPathRoute);
+                    MessageBox.Show(message, Catalog.GetString("Attention"), MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    EnableButtons();
+                    return;
+                }
+            }
+
+            // the download
+
+            dataGridViewDownloadContent.CurrentRow.Cells[1].Value = Catalog.GetString("Installing...");
+            Refresh();
+
+            if (!await Task.Run(() => downloadRoute(installPathRoute)))
+            {
+                EnableButtons();
+                return;
+            }
+
+            // insert row in Options, tab Content
+
+            if (!insertRowInOptions(installPathRoute))
+            {
+                EnableButtons();
+                return;
+            }
+
+            route.Installed = true;
+
+            DateTime dateTimeNow = DateTime.Now;
+            dataGridViewDownloadContent.CurrentRow.Cells[1].Value = dateTimeNow.ToString(CultureInfo.CurrentCulture.DateTimeFormat);
+            route.DateInstalled = dateTimeNow;
+
+            route.DirectoryInstalledIn = installPathRoute;
+
+            Settings.Folders.Save();
+            Settings.Content.Save();
+
+            MainForm mainForm = ((MainForm)Owner);
+            mainForm.LoadFolderListWhithoutTask();
```

Comment:
```suggestion
            mainForm.LoadFolderListWithoutTask();
```

Reply:
Just a typo

Reply:
Solved in https://github.com/openrails/openrails/pull/903/commits/7b5496d13bf0053a5a3c0b2700bb236424aeb3da